crates/contracts/src/alloy.rs

```diff
@@ -510,6 +546,23 @@ mod tests {
     fn test_has_address() {
         assert!(BaoswapRouter::deployment_address(&GNOSIS).is_some());
         assert!(HoneyswapRouter::deployment_address(&GNOSIS).is_some());
+
+        for chain_id in &[MAINNET, ARBITRUM_ONE, BASE, BNB] {
```

Comment:
I'm not sure it's worthwhile to port these tests over as I feel like they don't help much. It's not like we add and remove the deployment addresses for the same contract.